packages/components/molecules/f-restaurant-card/src/components/RestaurantCard.vue

```diff
@@ -426,7 +426,7 @@ export default {
 }
 
 .c-restaurantCard-imageTags {
-    bottom: spacing(b);
+    bottom: spacing(c);
```

Comment:
is this the same value as the logo?

Reply:
Yes and no, so the logo has a spacing of **d** which is 16px, while **c** is 12px, however the restaurant tag has a hardcoded 4px bottom margin, therefore it ends up being 16px, the hardcoded 4px is for when the tags are in tile view and when they wrap so that they don't touch each other

Reply:
It sounds like that 4px margin might need looking at. Are you sure that can't be moved somewhere else?

Reply:
I did think that, the alternative would be to remove the 4px and add a flex gap to the parent

Reply:
Updated 😄